e2e/config/config_test.go

```diff
@@ -0,0 +1,130 @@
+package config_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/ramendr/ramen/e2e/config"
+	"github.com/ramendr/ramen/e2e/types"
+)
+
+func TestValidatePVCSpecs(t *testing.T) {
+	// storageClass name exceeding 253 chars
+	longSCName := strings.Repeat("a", 254)
+
+	tests := []struct {
+		name    string
+		config  *types.Config
+		wantErr bool
+	}{
+		{
+			name: "empty PVCSpecs list",
```

Comment:
since the tests name is TestValidatePVCSpecs - we can use short name like "empty".